packages/core/src/app/shipping/ConsignmentLineItem.tsx

```diff
@@ -0,0 +1,111 @@
+import { ConsignmentLineItem } from "@bigcommerce/checkout-sdk";
```

Comment:
Not sure why our linter didn't pick this up. This file appears to have some eslint issues, like not using single quote.
I also observed that the linting rules were not applied consistently in `checkout-js`. 
Perhaps after this PR, we can investigate why the linter is behaving inconsistently and do a quick cleanup.
cc @animesh1987 

<img width="1208" alt="image" src="https://github.com/user-attachments/assets/bc00bf81-db01-4c41-8c17-c94ae313e34b">

Reply:
The rest looks good to me.

Reply:
Sounds good, maybe our config is incorrect.